Code/Framework/AzToolsFramework/Tests/ManipulatorBoundsTests.cpp

```diff
@@ -187,4 +187,51 @@ namespace UnitTest
         EXPECT_NEAR(intersectionDistance, 10.0f, g_epsilon);
         EXPECT_TRUE(intersection);
     }
+
+    // replicates a scenario in the Editor using a cone and a pick ray which should have failed but passed with Intersect::IntersectRayCone
+    TEST(ManipulatorIntersectRayConeTest, RayConeEditorScenarioTest)
+    {
+        auto rayOrigin = AZ::Vector3(0.0f, -0.808944702f, 0.0f);
+        auto rayDir = AZ::Vector3(0.301363617f, 0.939044654f, 0.165454566f);
+        float t = 0.0f;
+        bool hit = AzToolsFramework::Picking::IntersectRayCone(
+            rayOrigin, rayDir, AZ::Vector3(0.0f, 0.0f, 0.161788940f), AZ::Vector3(0.0f, 0.0f, -1.0f), 0.0453009047, 0.0113252262, t);
+        EXPECT_EQ(hit, false);
```

Comment:
Nit:  Could use EXPECT_TRUE / EXPECT_FALSE for a lot of these.

Reply:
Sure I can update these 👍